photo1.css

```diff
@@ -0,0 +1,253 @@
+body {
+  position: relative;
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.body__footer {
+  top: 765px;
+  margin-bottom: 25px;
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.bottomNavBar {
+  display: flex;
+  align-items: center;
+  width: 499.8px;
+  justify-content: space-between;
+  height: 15px;
+}
+
+.footer__link {
+  color: rgb(110, 110, 110);
+  text-decoration: none;
+}
+
+.footer {
+  display: flex;
+  color: rgb(110, 110, 110);
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-content: space-between;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  font-size: 10.0008px;
+  font-stretch: 100%;
+  font-weight: 400;
+  width: 980px;
+}
+
+.footer__bottomNavBar {
+  display: inline-flexbox;
+  float: right;
+}
+
+.footer__textSection {
+  float: left;
+  margin: 0;
+}
+
+.main__photo1 {
+  position: absolute;
+  margin: auto;
+  width: 980px;
+}
+
+.main__smallerPhotos {
+  position: absolute;
+  top: 540px;
+  width: 980;
+}
+
+.mainMenu {
+  width: 980px;
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  height: 36px;
+  padding: 0;
+  border-radius: 4px;
+  background-image: url(http://images.apple.com/global/nav/images/globalheader.png);
```

Comment:
The background gradient should be implemented using the `linear-gradient` property. Please remove the background image here.